docs/CHANGELOG.asciidoc

```diff
@@ -8,16 +8,2498 @@
+
+For detailed information about the 8.5.0 release, review the enhancements and bug fixes.
+
+[float]
+[[enhancement-v8.5.0]]
+=== Enhancements
+Alerting::
+* 141189 alerts table performance {kibana-pull}141385[#141385]
+* 141119 remove visibility toogle + use_columns refactor {kibana-pull}141250[#141250]
+* Adds Stats on top of execution logs {kibana-pull}140883[#140883]
+* Adds the Logs tab to Rules and Connectors UI {kibana-pull}138852[#138852]
+* Adds "exclude previous hits" check box to ESQuery rule form {kibana-pull}138781[#138781]
+* The ES Query Rule Type now supports Runtime Mappings and the Fields parameters when using an Elasticsearch DSL query {kibana-pull}138427[#138427]
+
+APM::
+* Service metrics on inventory page {kibana-pull}140868[#140868]
+* Sort trace samples {kibana-pull}140589[#140589]
+* Setting for tail sampling storage limit in Fleet integration {kibana-pull}140567[#140567]
+* Adds AWS Lambda metrics to "Metrics" tab UI {kibana-pull}140550[#140550]
+* Adds Experimental Mode for APM UI {kibana-pull}139553[#139553]
+* Rename JVMs to Metrics {kibana-pull}138437[#138437]
+* Display partial data buckets differently and message in the chart tooltips {kibana-pull}137533[#137533]
+
+Dashboard::
+Adds the ability to view panel-level filters and queries {kibana-pull}136655[#136655]
+
+Discover::
+* Adds the ability to only show reset height button when current height is not the default {kibana-pull}143013[#143013]
+* Adds the ability to persist histogram height to local storage {kibana-pull}140997[#140997]
+* Enables `Explore in Discover` for adhoc data views in *Lens* {kibana-pull}140726[#140726]
+* Adds the ability to show actions inline in the Expanded Document view for quick access {kibana-pull}140085[#140085]
+* Updates the layout for unified histogram {kibana-pull}139446[#139446]
+* Adds new field stats in sidebar popover {kibana-pull}139072[#139072]
+* Adds ad-hoc data views {kibana-pull}138283[#138283]
+* Updates the formatter for aggregate_metric_double field values {kibana-pull}138205[#138205]
+
+Elastic Security::
+For the Elastic Security 8.5.0 release information, refer to {security-guide}/release-notes.html[_Elastic Security Solution Release Notes_].
+
+Fleet::
+* Adds toggle for experimental synthetic `_source` support in Fleet data streams {kibana-pull}140132[#140132]
+* Enhances the package policy API to create or update a package policy API with a simplified way to define inputs {kibana-pull}139420[#139420]
+* Support new subscription and license fields {kibana-pull}137799[#137799]
+
+Infrastructure::
+* Adds log-* index pattern support on SM UI {kibana-pull}139121[#139121]
+* Adds health API support for data ingested from package {kibana-pull}138964[#138964]
+* Adds support for integration package {kibana-pull}138224[#138224]
+* Adds the inital hosts page {kibana-pull}138173[#138173]
+
+Lens & Visualizations::
+* Adds support for noPadding option to Lens embeddable {kibana-pull}140442[#140442]
```

Comment:
Can be removed, this is an internal thing - I adjusted the label on the PR already.